fitness/templates/account_settings.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'base.html' %}
+<html>
```

Comment:
getting closer, take a look at https://github.com/Davinci-Coders/workout_tracker/blob/master/fitness/templates/home.html

for examples of how to use `{% extends` and `{% block`

happy to add docstrings if they're not clear